metadata-io/metadata-io-api/src/main/java/com/linkedin/metadata/entity/ebean/batch/ChangeItemImpl.java

```diff
@@ -80,19 +84,44 @@ public static ChangeItemImpl fromPatch(
   @Nonnull private final AspectSpec aspectSpec;
 
   @Setter @Nullable private SystemAspect previousSystemAspect;
-  @Setter private long nextAspectVersion;
+  private long nextAspectVersion;
+  private final Map<String, String> headers;
+
+  @Override
+  public void setNextAspectVersion(long nextAspectVersion) {
```

Comment:
Can you help me understand what context is this used vs. `getSystemAspect`?
They both have version incrementing logic

Reply:
getSystemAspect allows overriding the version for some reason, I'd have to track down where and why. However since it overrides the version I just want to be sure it keeps the system metadata version in sync with whatever overriding is happening there